lib/ancestry/has_ancestry.rb

```diff
@@ -30,6 +30,8 @@ def has_ancestry options = {}
       validates_format_of self.ancestry_column, :with => derive_ancestry_pattern(options[:primary_key_format]), :allow_nil => true
       extend Ancestry::MaterializedPath
 
+      include Ancestry.optimizer if Ancestry.optimizer
```

Comment:
it may change in the future, but for now lets hardcode this thing

``
include Ancestry::MaterializedPathPg connection.adapter_name.downcase == "postgresql"
```

and for a test maybe make sure it has the correct ancestor.

```
if `ENV["DB"].to_s =~ /pg/i`
  model.ancestors.map(&:to_s).include?("Ancestry::MaterializedPathPg")
end
```

Reply:
hmm, [you sure about that line?](https://github.com/stefankroes/ancestry/blob/master/test/concerns/db_test.rb#L7)

Reply:
not marking this as resolved because i haven't gotten around to the feedback about the test either